src/CredentialSource/ExecutableSource.php

```diff
@@ -100,6 +100,17 @@ public function __construct(
         $this->executableHandler = $executableHandler ?: new ExecutableHandler();
     }
 
+    /**
+     * Gets the unique key for caching
+     * This source is considered non cacheable so returns null.
```

Comment:
the comment says "This source is considered non cacheable so returns null", but yet `getCacheKey` is returning something. We need to either fix the comment or have it return `null`.